plio/tes2mongo.py

```diff
@@ -0,0 +1,141 @@
+import sys
```

Comment:
Why is this in here? Otherwise, good to merge whenever. Looks great.

Reply:
omg this old as dirt. I think it's old code from capstone lol. I'll remove it